lib/diagnostic.js

```diff
@@ -8,22 +8,33 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
-
+  let result = [];
+  for (let i = 0; i < min.length; i++) {
```

Comment:
```min``` is the name of the function and its length is 0.  This condition evaluates to ```0 < 0```, which is false, so the loop never runs.